--- samba-mounter-0.2.orig/src/sambamount.desktop
+++ samba-mounter-0.2/src/sambamount.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Exec=systemsettings5 sambamount
+Exec=kcmshell5 sambamount
 Icon=preferences-system-network-sharing
 Type=Service
 
--- samba-mounter-0.2.orig/src/sambamounterapp.desktop
+++ samba-mounter-0.2/src/sambamounterapp.desktop
@@ -59,7 +59,7 @@ GenericName[uk]=Диски у мереж
 GenericName[x-test]=xxNetwork Drivesxx
 GenericName[zh_CN]=网络驱动器
 GenericName[zh_TW]=網路硬碟
-Exec=systemsettings5 sambamount
+Exec=kcmshell5 sambamount
 Comment=Network Samba Drives
 Comment[ca]=Unitats Samba de xarxa
 Comment[ca@valencia]=Unitats Samba de xarxa
